pkg/gorilla/gorilla.go

```diff
@@ -74,6 +76,21 @@ func (ws *WebSocket) SetTimeOut(timeout time.Duration) *WebSocket {
 	return ws
 }
 
+// If path is empty it will use os.stdout/os.stderr
+func (ws *WebSocket) Logger(path string) *WebSocket {
+	var err error
+	ws.logger, err = logger.NewLogger(path)
```

Comment:
This is the issue. Instead, we should have `logger.FromPath(path)` and `logger.FromBuffer(buffer)` or perhaps `logger.FromChannel(channel)`. We don't need the channel one so lets now add it, but hopefully it shows how that fits together. It great work overall though!

Reply:
I changed to a builder pattern it suits me better I think.
Also Updated the names of funds as you said.